droidconKE/chai
# Feature/font res uploads

---
name: Resources Upload
about: Uploads to some resources thaat w eneed to get started building some aspects of the code.
label: 'resources uploads'


---
## How can this PR be tested?
 Check the resources 🥇 

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:


# Screenshot
> For PRs that change the UI, attach a before/after screenshot.
> **[REMOVE THIS SECTION IF IT DOES NOT APPLY TO THIS PR]**

![]()
